data/secscan_model/secscan_v4_model.py

```diff
@@ -214,6 +214,28 @@ def needs_reindexing_query(indexer_hash):
         for candidate, abt, num_remaining in iterator:
             manifest = ManifestDataType.for_manifest(candidate, None)
             layers = registry_model.list_manifest_layers(manifest, self.storage, True)
+            if layers is None:
```

Comment:
Should we add a test for this?

Reply:
Yes, we should, but I wanted to get this fixed for QE. Should I do so as a followup or would you like to do so?

Reply:
We can do it as a followup.

Reply:
Added it to this PR